app/mbm/static/js/autocomplete.js

```diff
@@ -11,7 +11,7 @@ const canonicalComponents = [
 const initAutocomplete = (inputElement, markerName, app) => {
   // Create the autocomplete object
   let autocomplete = new google.maps.places.Autocomplete(
-    inputElement, { componentRestrictions: { country: "us" } }
+    inputElement, { componentRestrictions: { country: ["us", "ca"], } }
```

Comment:
This means the autocomplete options will include both US and Canada results for both the Chicago and Montreal pages. I think this is fine because they're sorted by proximity to the user anyway, but if we wanted to we could only show US results on the Chicago page and Montreal results on the Canada page.

Reply:
This thing: 

![CleanShot 2025-02-12 at 19 02 55@2x](https://github.com/user-attachments/assets/08b83911-44fa-4c10-9331-6253bd71aeb4)